README.md

```diff
@@ -8,14 +8,14 @@ A universal mechanism for calculating the estimated value of a collection of ass
 import {tami} from '@mimicry/tami';
 
 const transactions = [
-  { itemId: 'Lavender', timestamp: 1593129600000, price: 500 },
-  { itemId: 'Hyacinth', timestamp: 1600992000000, price: 700 },
-  { itemId: 'Hyacinth', timestamp: 1614211200000, price: 400 },
-  { itemId: 'Mars', timestamp: 1624406400000, price: 612 },
-  { itemId: 'Mars', timestamp: 1639008000000, price: 1200 },
+  { itemId: 'Lavender', timestamp: 1593129600000, price: 500, timestamp: new Date() },
```

Comment:
Here it looks like you have the timestamp values twice. Once as a hardcoded string and once as a new Date(). Is the addition of timestamp: new Data() needed?

Reply:
Fixed!